### DISCUSSION TITLE
Skolemisation of blank nodes

### DISCUSSION THREAD

**Discussion by syscharles** [afs/rdf-delta#233] [Q&A] [answered]
Hello, 
I am currently working on a project which contains a lot of blank nodes which I have to modify/delete sometimes.
Does RDF Delta offer somehow skolemisation of those blank nodes? 
Thanks in advance for your help!

**Comment by afs** [comment, marked as answer]
This project isn't a general RDF processing framework. It provides an additional facility for Apache Jena.

Apache Jena provides general RDF processing and can transform data.

The RDF patch format has moved to Apache Jena.
